run.cmd

```diff
@@ -1,60 +0,0 @@
-@if not defined _echo @echo off
```

Comment:
Should the `config.json` file be deleted as well?

Reply:
yes.